### ISSUE TITLE
[1906.03361] Robust Bi-Tempered Logistic Loss Based on Bregman Divergences

### ISSUE THREAD

**Issue by riron1206** [riron1206/read_paper#42]
## 概要
https://arxiv.org/abs/1906.03361
Bi-Tempered Logistic Loss
ノイズの多いデータでニューラルネットをトレーニングするための損失関数
バイテンパーは大きな外れ値にも小さな外れ値にもロジスティック損失より良く対応できる

バイテンパーは、2つの調整可能な「温度(temperatures)」パラメーターを使う事で、ロジスティック損失が持つ問題をうまく処理
- t1は損失関数の有界性(つまり、マージンの最大値)
- t2は裾の重さ(つまり、伝達関数のテール部の減衰率)

t1とt2の両方を1.0に設定すると、ロジスティック損失関数と同じ
t1<1.0だと決定境界が狭まる
t2>1.0だと、伝達関数(softmax等)の裾が広くなる

左図：t1によりラベルが間違っていても、各例で発生するlossをそのまま渡さず軽減する
右図：t2により確率値を多めに割り当てる
![imag1](https://1.bp.blogspot.com/-MSsW9QUCyXM/XWQNInb2ztI/AAAAAAAAEjc/_yu48eAnTjMWQtUbjNqhOlqurgapjSIzACLcBGAs/s1600/Screenshot%2B2019-08-26%2Bat%2B9.39.48%2BAM.png)

白い線が決定境界
![imag0](https://1.bp.blogspot.com/-wTnN2w3ENVc/XWQOEFA67TI/AAAAAAAAEjo/0W-4vcYefTcwn7tURpmQ6zMUBrUmJ5e-wCLcBGAs/s1600/Screenshot%2B2019-08-26%2Bat%2B9.37.41%2BAM.png)

ノイズの多いデータにニューラルネットワークが対応する能力は損失関数に大きく依存する
分類タスクで良く使われる標準的な損失関数はロジスティック損失(softmax)だが外れ値に敏感なため2つの問題あり
- 大きな外れ値が全体的な損失を支配する可能性あり
- 誤ったラベルが、決定境界を広げてしまう可能性あり

※ノイズは、破損した例(猫の画像にレンズフレアが写り込むなど)、データが収集されたときのラベルの誤った例(スパイダーマンの画像に「蜘蛛」と誤ったラベル付けがされるなど)

## コメント
- 公式のgithub: 
    - https://github.com/google/bi-tempered-loss (TF)

<br>

- 非公式実装
    - https://github.com/mlpanda/bi-tempered-loss-pytorch (pytorch)

<br>

- デモサイト
    - https://google.github.io/bi-tempered-loss/

<br>

- 参考サイト
    - https://ai.googleblog.com/2019/08/bi-tempered-logistic-loss-for-training.html
    - https://webbigdata.jp/ai/post-4062
    - https://webbigdata.jp/ai/post-4068
    - https://webbigdata.jp/ai/post-4074
